app/javascript/mastodon/features/ui/index.js

```diff
@@ -350,6 +355,12 @@ class UI extends React.PureComponent {
       navigator.serviceWorker.addEventListener('message', this.handleServiceWorkerPostMessage);
     }
 
+    // On first launch, redirect to the follow recommendations page
+    if (this.props.firstLaunch) {
+      this.context.router.history.replace('/start');
+      this.props.dispatch(closeOnboarding());
```

Comment:
Not completely sure about this. What if a user mistakenly or hastily closes the page without looking at the onboarding screen, and never see it again?

Reply:
I could tie it to `componentWillUnmount` but ultimately I don't think that would change anything. I want people to be able to straight to search and not have the screen show up next time confusingly because they haven't pressed done.